internal/httphandler/config_request_handler.go

```diff
@@ -29,7 +30,7 @@ type configRequestHandler struct {
 }
 
 // NewConfigRequestHandler return config query and transactions request handler
-func NewConfigRequestHandler(db bcdb.DB, logger *logger.SugarLogger) http.Handler {
+func NewConfigRequestHandler(db bcdb.DB, logger *logger.SugarLogger, metricsRegistry *prometheus.Registry) http.Handler {
```

Comment:
That is relevant to all handlers - we should probably start with data and move on from there.